### DISCUSSION TITLE
Installed in wrong folder

### DISCUSSION THREAD

**Discussion by Dejon** [o3de/o3de#10337] [Help Wanted] [answered]
I've just installed O3DE for the first time and I kept waiting for the option to customize my install. I didn't receive a prompt so the whole thing installed to my C drive. Is there any way to change this? Thanks

**Comment by AMZN-alexpete** [comment, marked as answer]
Yes @Dejon you just need to click the "Options" button on the first page to change the location.  

![image](https://user-images.githubusercontent.com/26804013/177888854-b793b15c-8ce2-4766-930a-edd0f9181341.png)

We have a task to improve the installer so the install location is visible by default and we have better downloading progress information.

**Reply by Dejon** [reply]
Thanks. I was beginning to wonder if anyone was home :)
